src/ExpenseTrackerGrupo2/DataAccess/Concretes/ExpenseRepository.cs

```diff
@@ -0,0 +1,26 @@
+using Dapper;
+using ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.Data;
+using ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.DataAccess.Entities;
+
+namespace ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.DataAccess.Concretes;
+
+public class ExpenseRepository : BaseRepository<Expense>, IExpenseRepository
+{
+    public ExpenseRepository(IDbConnectionFactory dbConnectionFactory) : base(dbConnectionFactory) { }
+
+    public async Task<IList<Expense>> GetExpenseByCategory(string category)
+    {
+        using var connection = await _dbConnectionFactory.CreateConnection();
+        var query = "SELECT * FROM Expenses WHERE Category = @Category";
```

Comment:
@gabriela-garcia-jalafund 
Don't forget to update the table and column name for query access.